### PR TITLE
Update windows release to use 22.03 base sw 

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: Dockerfile.win10.min [hunk 1]
```diff
@@ -92,10 +92,10 @@ ARG CUDA_PACKAGES="nvcc_${CUDA_MAJOR}.${CUDA_MINOR} \
                    visual_studio_integration_${CUDA_MAJOR}.${CUDA_MINOR}"
 ARG CUDA_INSTALL_ROOT_WP="C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v${CUDA_MAJOR}.${CUDA_MINOR}"
 
-ARG TENSORRT_VERSION=8.2.2.1
+ARG TENSORRT_VERSION=8.2.4.2
 ARG TENSORRT_ZIP=TensorRT-${TENSORRT_VERSION}.Windows10.x86_64.cuda-11.4.cudnn8.2.zip
```
Comment (rmccorm4) [inline Dockerfile.win10.min:96]: I know this change isn't in this PR, but just double checking that this TENSORRT_ZIP URL is fine to use with CUDNN 8.3 and CUDA 11.5 even though the link here is `*.cuda-11.4.cudnn8.2.*` ? Should the TRT ZIP be updated to CUDA 11.5 and CUDNN 8.3 if available?

**Review by rmccorm4 — APPROVED** [review]
LGTM other than one question

**Review by rmccorm4 — COMMENTED** [review]
I know this change isn't in this PR, but just double checking that this TENSORRT_ZIP URL is fine to use with CUDNN 8.3 and CUDA 11.5 even though the link here is `*.cuda-11.4.cudnn8.2.*` ? Should the TRT ZIP be updated to CUDA 11.5 and CUDNN 8.3 if available?

File: Dockerfile.win10.min (hunk 1 above)
Comment (mc-nv) [reply Dockerfile.win10.min:96]: Yes, it also bring my attention at the time I'm source the dependencies.
But you can review the link evaluation on below screenshot: 
image

Comment (rmccorm4) [reply Dockerfile.win10.min:96]: I see, LGTM - thanks!

**Review by rmccorm4 — DISMISSED** [review]

**Review by tanmayv25 — APPROVED** [review]

**Review by GuanLuo — APPROVED** [review]

**Comment by mc-nv** [comment]
Closing back to review, I was unable to bypass testing.

**Pull request closed without merging**